cpp/lib/include/IssnLookup.h

```diff
@@ -38,9 +38,9 @@ struct ISSNInfo {
         type_,               // type
         issn_,               // issn
         is_part_of_,         // isPartOf
-        publication_,        // publication
-        url_;                // url
-    std::vector<std::string> names_;
+        publication_;        // publication
+
+    std::vector<std::string> names_, url_;
```

Comment:
If we name a variable and it may contain multiple values, it's good to name it with the plural form
url_ => urls_